### PR TITLE
Move grpcio_tools to src/python

### PR DESCRIPTION
**Pull request by soltanmm-google**
Alright let's get this over with.

### CODE REVIEW

File: tools/dockerfile/bazel/Dockerfile [hunk 1]
```diff
@@ -32,21 +32,20 @@ RUN apt-get update
 RUN apt-get -y install software-properties-common python-software-properties
```
Comment (kpayson64) [inline tools/dockerfile/bazel/Dockerfile]: Can we remove these changes from this PR?  Or are they required?

Comment (soltanmm-google) [inline tools/dockerfile/bazel/Dockerfile]: They could be moved to another PR, but they're necessary (`make_grpcio_tools.py` does the wrong thing with the dockerfile being broken).

Comment (kpayson64) [inline tools/dockerfile/bazel/Dockerfile]: That's fine then

**Comment by soltanmm-google** [comment]
Build to follow: https://grpc-testing.appspot.com/view/Artifacts/job/gRPC_build_artifacts/2621/

**Comment by soltanmm** [comment]
Fixes #9413. Once conflicts are resolved, anyway.

**Comment by soltanmm** [comment]
(and the build failure is a known Python 3.5 error)

**Comment by nathanielmanistaatgoogle** [comment]
What do we think of targeting this to the `v1.1.x` branch? Over the life of `v1.0.x` I felt like we were happiest with "put everything on `v1.0.x` and let it get upmerged to `master`".

**Comment by soltanmm** [comment]
I'll probably get around to that over the weekend.

**Comment by soltanmm-google** [comment]
#9498
